src/validation-group.js

```diff
@@ -92,7 +92,7 @@ export class ValidationGroup {
    * Causes complete re-evaluation: gets the latest value, marks the property as 'dirty' (unless false is passed), runs validation rules asynchronously and updates this.result
    * @returns {Promise} A promise that fulfils when valid, rejects when invalid.
    */
-  validate(forceDirty = true, forceExecution = true) {
+  validate(forceDirty = true, forceExecution = true): Promise<ValidationResult> {
```

Comment:
currently babel-dts-generator doesn't infer that default `true` means that parameter type is `boolean` 
(result produces 
`validate(forceDirty?: any, forceExecution?: any): Promise<ValidationResult>;`), instead of 
`validate(forceDirty?: boolean, forceExecution?: boolean): Promise<ValidationResult>;`)
So that correct would be to replace this line with `validate(forceDirty: boolean = true, forceExecution: boolean = true): Promise<ValidationResult> {`